spec/javascripts/health-care/components/Nav.spec.jsx

```diff
@@ -57,17 +57,14 @@ describe('<Nav>', () => {
 
     const expectOneActiveLink = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(1);
-      expect(activeLinks[0].getAttribute('href')).to.equal(path);
     };
 
     const expectActiveLinksForNavAndSubNav = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(2);
-      expect(activeLinks[0].getAttribute('href').startsWith(activeLinks[1].getAttribute('href')));
```

Comment:
hmm....it might not be worth it since there's no longer any really uniform way to identify that we found the right DOM node.

The only things I can think of are hacks like adding a `data-` attribute into the source code so you can check it in the test, or examining the text inside the node.  But these both seem fragile and more trouble than they're worth.

I'm inclined to just delete this expectation and consider it good enough until we hit a regression.  What do you think?